docs/api-docs/TypeScript.md

```diff
@@ -0,0 +1,34 @@
+# `Typescript`
+
+## Usage with `react-redux`
+
+In order to use `redux-loop` within a  TypeScript React project, you'll need to import the type
```

Comment:
I was thinking that maybe they could directly pull from `node_modules` in the `tsconfig.json`. But maybe that would be bad practice - I suggested to try that where we implement this to see how it looks/feels. It feels lame to have to make a file just to have a single line import.. But that is *probably* better so that node figures out how to resolve stuff rather than us hardcoding the module's file system path in `tsconfig.json`. What do you guys think @DanielOrtel @bdwain ?

Reply:
there are a lot of different package managers and with `pnp`, this wouldn't be possible. I think the docs should show a solution that works regardless of the package manager.

Reply:
Ah, that's a good point.